### ISSUE TITLE
missing riak_get_response accessors

### ISSUE THREAD

**Issue by jrwest** [basho-labs/riak-c-client#14]
Functions to access the fields of `riak_get_response` should be added. Otherwise, unless only printing the response, `riak_messages-internal.h` must be included to access the individual fields:

```
error: dereferencing pointer to incomplete type
     if (get_resp->n_content == 1) {
```

**Comment by hazen** [comment]
@jrwest LOTS of accessors have not been added yet.  Good to track this shortcoming, however.

**Comment by hazen** [comment]
Resolved by #27
